modules/node-network-configuration-policy-file.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-network-configuration-policy-file.adoc_{context}"]
+= The NodeNetworkConfigurationPolicy manifest file
+
+A `NodeNetworkConfigurationPolicy` (NNCP) manifest file defines policies that the Cluster Network 
+Operator (CNO) uses to configure networking for nodes that exist in an {product-title} cluster. 
```

Comment:
Change "Cluster Network Operator" to "Kubernetes NMstate Operator"

Reply:
:heavy_check_mark: